### DISCUSSION TITLE
error in the bounding boxes detection

### DISCUSSION THREAD

**Discussion by yashathalye** [roboflow/notebooks#46] [Q&A]
Saw the Track & Count Objects using YOLOv8 ByteTrack & Supervision video and tried with my use case but i am getting the error as "Bounding boxes batch must be defined as 2d np.array with (N, 4) shape, [] given" in the tqdm section your help is much obliged thank you

**Comment by mikearney** [comment]
I got the same error on that tutorial and had to abandon it

**Reply by yashathalye** [reply]
i think the error is occurring due the weights file
i created the weights file in yolov8-s model and the code is for -x model

**Reply by SkalskiP** [reply]
@yashathalye, @mikearney, thanks a lot for bringing it to my attention. It is because, on your video, there are most likely frames when there are no detected objects. And that method is not expecting that to happen. This is, of course, my fault.

First, let me convert this discussion topic into a bug report. Second I'll try to fix our notebook right away. @mikearney please don't lose your trust in this tutorial. 🙏🏻

**Comment by yashathalye** [comment]
Thanks for replying @SkalskiP the program successfully one time and counting was also better but when I tried on a different video it gave a different error about cannot import ultralyticsCFG .

**Reply by SkalskiP** [reply]
@yashathalye, @mikearney I think I managed to solve the original problem: `Bounding boxes batch must be defined as 2d np.array with (N, 4) shape, [] given`. Could any of you try to run it and confirm that the problem no longer exists? 

@yashathalye as for `cannot import ultralyticsCFG`. Could you give me a bit more details? When / where is that issue happening?

**Comment by yashathalye** [comment]
Thanks a lot @SkalskiP  the issue for Bounding boxes batch must be defined as 2d np.array with (N, 4) shape, [] given is solved!!!! and the counting is accurate. as for the ultralyticsCFG ERROR here is the screenshot.
![Egg_tracking_counting_byte_tracker ipynb - Colaboratory - Google Chrome 23-01-2023 14_32_28](https://user-images.githubusercontent.com/91899309/214209830-6eecc5a1-5f63-416f-a1fc-e5476215d831.png)

the error is occuring in the line 

from ultralytics import YOLO
 **_model = YOLO(MODEL)_**

**Reply by SkalskiP** [reply]
@yashathalye did you use a pre-trained model or your custom one?

**Reply by yashathalye** [reply]
Yes sir i used the YOLOv8-s object detection model to create weights file. and i exported yolov8 api-key from roboflow

**Reply by SkalskiP** [reply]
and do you know what was the version of `ultralytcs` pip package in the notebook that you used for training?
